homeassistant/components/venstar/sensor.py

```diff
@@ -52,6 +84,15 @@ async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
             ]
         )
 
+    runtimes = coordinator.runtimes[-1]
+    for sensor_name in runtimes.keys():
```

Comment:
```suggestion
    for sensor_name in runtimes:
```

Reply:
This is a dict, not a list.  coordinator.runtimes gets back a list of dictionaries, of which we always want the last one.

Reply:
Iterating a dict iterates over the keys by default. Try it out.